backend/src/packages/users/user.model.ts

```diff
@@ -31,11 +31,11 @@ class UserModel extends AbstractModel {
     return {
       withoutPassword(builder): QueryBuilder<UserModel> {
         return builder.select(
```

Comment:
what is returned after this query? don't we need to add other fields like surveyIsCopmleted etc

Reply:
or are they added automatically?

Reply:
They are added in the findById method in the user.repo

Reply:
As I understand, it means that it can be quite misunderstanding that we don't return other required fields there. We got them indeed, but can't see that in code

Reply:
well, modifier was added to make sure that password information wouldn't be seen where it;s not needed at all. But overall in the findById method the Graph is joined with the user details, where we get all extra information

```
 const user = await this.userModel
      .query()
      .modify('withoutPassword')
      .withGraphJoined(UsersRelation.DETAILS)
      .findById(id)
      .castTo<UserCommonQueryResponse | undefined>()
      .execute();
```